### DISCUSSION TITLE
OSError: models\x does not appear to have a file named config.json

### DISCUSSION THREAD

**Discussion by jasonmhead** [oobabooga/text-generation-webui#892] [Q&A]
When the error is like
```OSError: models\RWKV does not appear to have a file named config.json```
or
```OSError: models\neox-ckpt-pythia-70m does not appear to have a file named config.json```

and huggingface doesn't have a config.json in that repo, what is the best way to fix this?

I'm thinking RWKV might work differently than other setups...

**Comment by Desusaur** [comment]
Did you ever get this figured out?
